lib/directories.zsh

```diff
@@ -24,10 +24,11 @@ alias rd=rmdir
 alias d='dirs -v | head -10'
 
 # List directory contents
-alias lsa='ls -lah'
-alias l='ls -lah'
+# alias lsa='ls -lah'
+alias l='ls'
```

Comment:
I don't think the persistence of old `ls` aliases (which looks a bit of messy) is reasonable, since the oh-my-zsh project doesn't reach the final version now.
And, some popular Linux distribution has already aliased the `ls` to `l` , `la` to `ls -a`, such as the Ubuntu.